tests/unit/serialization/binary_compatibility/reference_objects.py

```diff
@@ -654,6 +704,37 @@ def __eq__(self, other):
     ),
     "BigInteger": 1314432323232411,
     "BigDecimal": decimal.Decimal("31231.12331"),
+    "BigDecimal[]": [
```

Comment:
I think we shouldn't define them here. These are not part of the reference objects (which is the equivalent of `allTestObjects` in Java) We can define them outside, just like we did for things like `_data ` below

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/pull/592/commits/a5ac0727172f5527fae403c60cb2fab2426c360a . Thanks for the clarification, I was desperately excluding them manually 😄